tests/storage/test_throughput.py

```diff
@@ -18,10 +19,13 @@ def get_args():
     parser.add_argument("--num", dest="number", type=int,
                         required=True,
                         help='Number of files to be generated')
+    parser.add_argument("--path", dest="path",
```

Comment:
I understand your point here but this might make the whole thing too much complicated.
I do not have a better solution for now, though.